docs/contributor/benchmarks/otlp-logs-validation.md

```diff
@@ -0,0 +1,407 @@
+- `receivers/filelog/operators`: The copy body to `attributes.original` must be avoided if `dropLogRawBody` flag is enabled.
+
+### How does checkpointing work
+
+> By enabling the storeCheckpoint preset (Helm) the `file_storage` extension is activated in the receiver
+> - The `file_storage` has the path `/var/lib/otelcol`
+> - This path is later mounted as a `hostPath` volume in the DaemonSet spec
+> - The extension is also set in the `storage` property of the filelog receiver
+
+> `storage` = The ID of a storage extension to be used to store file offsets. File offsets enable the receiver to pick up where it left off in the case of a collector restart. If no storage extension is used, the receiver manages offsets only in memory.
+
+
+## Benchmarking Setup
+
+1. Apply the configuration (with Prometheus):
+    ``` bash
+    k create ns prometheus
+    helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
+    helm repo update
+    helm upgrade --install -n "prometheus" "prometheus" prometheus-community/kube-prometheus-stack -f hack/load-tests/values.yaml --set grafana.adminPassword=myPwd
+
+    k apply -f telemetry-manager/hack/load-tests/log-agent-test-setup.yaml
+    ```
+
+2. To execute the load tests, the generated logs must be isolated. Replace the following line in the ConfigMap of the log agent:
+
+    ``` yaml
+    receivers:
+      filelog:
+        # ...
+        include:
+        - /var/log/pods/*/*/*.log # replace with "/var/log/pods/log-load-test*/*flog*/*.log"
+    ```
+
+3. If you want to run the backpressure scenario, additionally apply:
+    ``` bash
+    k apply -f telemetry-manager/hack/load-tests/log-backpressure-config.yaml
+    ```
+
+4. PromQL Queries used for measuring the results:
+    ``` sql
+    -- RECEIVED
+    round(sum(rate(otelcol_receiver_accepted_log_records{service="telemetry-log-agent-metrics"}[20m])))
+
+    -- EXPORTED
+    round(sum(rate(otelcol_exporter_sent_log_records{service="telemetry-log-agent-metrics"}[20m])))
+
+    -- QUEUE
+    avg(sum(otelcol_exporter_queue_size{service="telemetry-log-agent-metrics"}))
+
+    -- MEMORY
+    round(sum(avg_over_time(container_memory_working_set_bytes{namespace="kyma-system", container="collector"}[20m]) * on(namespace,pod) group_left(workload) avg_over_time(namespace_workload_pod:kube_pod_owner:relabel{namespace="kyma-system", workload="telemetry-log-agent"}[20m])) by (pod) / 1024 / 1024)
+
+    -- CPU
+    round(sum(avg_over_time(node_namespace_pod_container:container_cpu_usage_seconds_total:sum_irate{namespace="kyma-system"}[20m]) * on(namespace,pod) group_left(workload) avg_over_time(namespace_workload_pod:kube_pod_owner:relabel{namespace="kyma-system", workload="telemetry-log-agent"}[20m])) by (pod), 0.1)
+    ```
+
+
+## Performance Tests Results
+
+### 📊 Benchmarking Session #1
+
+| Icon | Meaning                                              |
```

Comment:
Are these icons only relevant for `Benchmarking Session #1`?

If they're relevant for all sessions, they shouldn't be under the `Session #1` headline. Instead, you could put them right below the parent headline (line 104).